maven-core/src/main/java/org/apache/maven/execution/DefaultBuildResumptionDataRepository.java

```diff
@@ -105,7 +109,7 @@ public void removeResumptionData( MavenProject rootProject )
     private Properties loadResumptionFile( Path rootBuildDirectory )
     {
         Properties properties = new Properties();
-        Path path = Paths.get( RESUME_PROPERTIES_FILENAME ).resolve( rootBuildDirectory );
+        Path path = rootBuildDirectory.resolve( Paths.get( RESUME_PROPERTIES_FILENAME ) );
```

Comment:
You can use `Path.resolve(String)` instead of first explicitly converting the argument to Path.

Reply:
Thanks, didn't know that!